src/storages/types.ts

```diff
@@ -355,6 +356,14 @@ export interface IImpressionCountsCacheSync extends IRecorderCacheProducerSync<R
   pop(toMerge?: Record<string, number> ): Record<string, number> // pop cache data
 }
 
+export interface IUniqueKeysCacheBase extends IRecorderCacheProducerSync<{ [featureName: string]: ISet<string> }>{
```

Comment:
This is the same than `IImpressionCountsCacheSync`?

If it is going to be revisited/fixed in another PR, please add a `TODO` comment next time.